src/extension-command-handler.ts

```diff
@@ -1,61 +1,65 @@
-/*
- * Handles commands from the client about tabs.
- */
-
-export default class TabHandler {
-  async COMMAND_TYPE_CREATE_TAB(_data: any): Promise<any> {
-    chrome.tabs.create({});
-  }
-
-  async COMMAND_TYPE_DUPLICATE_TAB(_data: any): Promise<any> {
+export default class ExtensionCommandHandler {
+  private async changeTab(direction: number): Promise<any> {
```

Comment:
if you make this a `Promise<void>`, then I think you can just do `resolve()` rather than `resolve(null)`